Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM python:3.10-alpine
+FROM python:alpine3.19 as builder
 
 WORKDIR /app
 
 COPY . .
 
-RUN pip install -r requirements.txt
+RUN pip install --user -r requirements.txt
```

Comment:
You have 3 RUN layers optimize them to have only 1 RUN layer

Reply:
1 RUN layer is at the builder stage, no idea to combine it with the other 2